gremlin-go/README.md

```diff
@@ -136,6 +136,16 @@ Note: The exact import name as well as the module prefix for `NewDriverRemoteCon
 		})
 ```
 
+## Aliases
+To make the code more readable and close to the Gremlin query language), you can use aliases
+```go
+	var __ = gremlingo.T__
+	var gt = gremlingo.P.Gt
+	var desc = gremlingo.Desc
```

Comment:
I find it slightly odd that this results in a mixture of capitalized and lower-case GLV steps. It makes sense that it would happen given the nature of Go, but think it's worth pointing out.

Maybe a little blurb of text would help. Something along the lines of "these aliases can be named with capital letters to be consistent with non-aliased steps but will result in exported variables which could be problematic if not being used in a top-level program (i.e. not a redistributable package)

Reply:
Second this.

Reply:
added in https://github.com/lyndonbauto/tinkerpop/pull/107/commits/37df33c58afc5f26654b0be4713b0c8efec675a2